assets/css/static-main.scss

```
@@ -148,6 +148,21 @@ article h2{
   .topmenu.active {
     border-radius: 3px;
   }
+  .benefit-prop img {
+    max-width: 50%;
+    display: block;
+    margin: 0 auto;
+  }
+  .api-card img {
+    max-width: 50%;
+    display: block;
+    margin: 0 auto;
+    min-height: 250px;
+  }
+  .api-card {
+      min-height: 500px;
+      width: 100%;
+  }
```

```suggestion
  .api-card {
    min-height: 500px;
    width: 100%;
    img {
      max-width: 50%;
      display: block;
      margin: 0 auto;
      min-height: 250px;
    }
  }
```
This isn't required at all, but because these are SCSS files, this could be rewritten in a nested structure. This _should_ do the same thing you have currently.